### DISCUSSION TITLE
Clarification Needed: Origin of Requests - Deployed Server or User IP Addresses?

### DISCUSSION THREAD

**Discussion by zhugw** [mitmproxy/mitmproxy#6334] [Q&A]
If mitmproxy is deployed on a server, and different individuals connect to the mitmproxy on the server as a proxy using their mobile devices to access a target website, I would like to know whether the requests to the target website appear to originate from the deployed server or from each user's individual IP address.

**Comment by mhils** [comment]
They appear to originate from the deployed server.

**Reply by zhugw** [reply]
Thanks! Due to strict risk control measures implemented by a certain e-commerce website, there are limitations on the number of times an account can access the website. Therefore, I am considering this crowdsourcing approach to collect more data. However, I am concerned that using the same IP address repeatedly might lead to being blocked.
